.github/workflows/on_push.yml

```diff
@@ -105,3 +111,89 @@ jobs:
         --container api-container \
         -- /opt/app/bin/liquid_voting \
         eval "LiquidVoting.Release.migrate"
+
+  teardown1:
```

Comment:
We don't need a teardown before. I think I see your concern that manual testing could break the CI if not properly cleaned up. I think probably the best way to address this is to restrict usage of this org_id to the CI, and any manual tests can be done with the demo org id. Or we can create separate sandbox org ids on demand for manual testing